### PR TITLE
Cleanup kubemark job configs - remove defaults and auto-calculated vars

### PR DESCRIPTION
**Pull request by shyamjvs**
Following https://github.com/kubernetes/kubernetes/pull/53974

/cc @wojtek-t

### CODE REVIEW

**Comment by shyamjvs** [comment]
/hold
For the other PR to be merged.

**Comment by shyamjvs** [comment]
/hold cancel
It's merged.

@wojtek-t Could you PTAL?

File: jobs/env/ci-kubernetes-kubemark-gce-scale.env [hunk 1]
```diff
@@ -1,31 +1,17 @@
...
-# Configuration we are targetting in 1.6
-STORAGE_MEDIA_TYPE=application/vnd.kubernetes.protobuf
-# The kubemark scripts build a Docker image
-JENKINS_ENABLE_DOCKER_IN_DOCKER=y
 
-# TODO: revert after running experiments.
-EVENT_PD=true
-# TODO: Reduce this once we have log rotation in Kubemark.
-KUBEMARK_MASTER_ROOT_DISK_SIZE=100GB
+### kubernetes-env
```
Comment (wojtek-t) [inline jobs/env/ci-kubernetes-kubemark-gce-scale.env]: Why did you remove MASTER_SIZE, MASTER_DISK_SIZE?

Comment (shyamjvs) [reply jobs/env/ci-kubernetes-kubemark-gce-scale.env]: They'd be auto-calculated based on NUM_NODES.
Ref: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-test.sh#L32-L34

**Comment by shyamjvs** [comment]
@wojtek-t Friendly ping :)
